src/app/components/apparatus-entry/apparatus-entry-readings/apparatus-entry-readings.component.html

```diff
@@ -10,7 +10,7 @@
     </ng-container>
```

Comment:
Why do you change this file?

Reply:
Only a minor style adjustment

Reply:
Even if it's not mine, I'm also removing exceding spaces here like you requested in the other templates